cmd/mdatagen/internal/samplereceiver/internal/metadata/generated_logs.go

```diff
@@ -23,7 +23,7 @@ type LogBuilderOption interface {
 	apply(*LogsBuilder)
 }
 
-func NewLogsBuilder(settings receiver.Settings) *LogsBuilder {
+func NewLogsBuilder(lbc LogsBuilderConfig, settings receiver.Settings) *LogsBuilder {
```

Comment:
I don't think we need to render the first argument if no events defined

Reply:
This is because I added an events configuration in the `samplereceiver` here: https://github.com/open-telemetry/opentelemetry-collector/pull/12822/files#diff-1fc56df0aa1a437b919d369e25fefa5bc78be2956cc2d507fbb7a0d1f0ca58d1R109